src/atomizer.js

```diff
@@ -678,11 +684,6 @@ Atomizer.prototype.getCss = function (config/*:AtomizerConfig*/, options/*:CSSOp
         }
     });
 
-    if (options.namespace) {
-        var jssNew = {};
-        jssNew[options.namespace] = jss;
-        jss = jssNew;
-    }
     if (options.helpersNS) {
```

Comment:
I think we decided long ago to **not** namespace helpers, but your reply to @src-code confused me; because you answered:

> Yes, for now.

to

> So Helpers always get namespaced? Even if the Helper is part of a contextual selector?

which to me suggests that helpers **do** get namespaced, no?

but then what does this mean exactly?

> Helpers are not namespaced today by default, only if you pass a namespace.

Can we pass a namespace to helpers only?
I'm confused...